### DISCUSSION TITLE
Feature Request: Language Selector Block selectable Language Flags and Sorting

### DISCUSSION THREAD

**Discussion by 6xthFNGR** [PrestaShop/PrestaShop#40251] [Ideas]
Hello,

many people like to add small Flag Icons next to the Language in the Language Selector Block.
There are several manuals in the PS Forum and other blogs, stating how to modify PS 1.6 and PS 1.7.
Like: https://www.themebooster.com/prestashop-tutorials/prestashop-1-7-and-1-6-display-language-flags/
or
https://www.prestashop.com/forums/topic/342578-how-to-add-flags-on-languages-block-for-ps-16/

As these modifications are not native, and are corrupted with updates, I would like to see this feature comes with PS already.

It would be great to have a "Configuration" Area of the Module, so the User can simply turn the flag icons on or off as he desires.
In the same BO Configuration, it would be great to decide in which order to display the languages: Ascended or Descending by Alphabetic, or a custom setting where you can position every language individual - like with the Module "Main Menu" where you can change the positions of the selected items.

Thanks

[a comment by an automated account is not shown]

**Comment by sowbiba** [comment]
Thanks for your suggestion @6xthFNGR 
I will let @PrestaShop/prestashop-product-team take a look at the need and come back to you asap

Dont hesitate to add screenshots of what you expect to be more clear
